Gemfile

```diff
@@ -38,7 +38,7 @@ group :development, :test do
   gem 'launchy'
   gem 'byebug'
   gem 'pry'
-  gem 'mocha'
+  # gem 'mocha'
```

Comment:
you mentioned this doesn't work with rspec so we should go ahead and just remove this line from the project